src/actplayer.cpp

```diff
@@ -3033,7 +3033,6 @@ void actPlayer(Entity* my)
 		if ( ticks % 4 == 0 )
 		{
 			int entityLight = my->entityLightAfterReductions(*stats[PLAYER_NUM], my);
-			entityLight = std::max(TOUCHRANGE, entityLight);
```

Comment:
If entityLightAfterReductions gave a value less than 32, this test / debug radius wouldn't show you. SAD!